javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,48 @@
+//Student manager
+const class07Students = [];
+function addStudentToClass(studentName)
+{
+    if(studentName === "")
```

Comment:
what if the function is called like this `addStudentToClass(null)` or `addStudentToClass()`?